slicetag.sh

```diff
@@ -48,7 +48,7 @@ set -e
 command=$1
 shift || :
 
-TAG=$( git rev-list --all | wc -l ) 
+TAG=$( date -u +"%s" ) 
```

Comment:
Is the issue of incremental versions getting out of sync between the master & forked repos unique to the timestamp approach? I think I can imagine scenarios where this would apply even with the commit-count approach.

The more general issue you're observing is that we still have to think too much about where we build from to guarantee an issue-free test-to-production cycle. This is necessary to think about. My thought is that this PR needs to solve it right now.